cmd/util/client.go

```diff
@@ -51,3 +57,71 @@ func ParseAWSTags(tags []string) (map[string]string, error) {
 	}
 	return tagMap, nil
 }
+
+func GetAMI(ctx context.Context, pullSecretFile string, instanceType string, region string, image string) (string, error) {
```

Comment:
See https://github.com/openshift/hypershift/pull/1594/files#r926762974
If we want to we can have a CLI to populate the type